build.sh

```diff
@@ -269,6 +323,10 @@ for DOCKER_TARGET in "${DOCKER_TARGETS[@]}"; do
         --label "NETBOX_GIT_URL=${NETBOX_GIT_URL}"
       )
     fi
+    if [ -n "${BUILD_REASON}" ]; then
+      BUILD_REASON=$(sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' <<< "$BUILD_REASON")
+      DOCKER_BUILD_ARGS+=( --label "BUILD_REASON=${BUILD_REASON}" )
```

Comment:
What is your idea that you add this as a label? Wouldn't it be enough to just print it to the console?

Reply:
I thought it would be use full to know why a particular image was build without searching through the build logs.